xchange-stream-ftx/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.knowm.xchange</groupId>
+        <artifactId>xchange-parent</artifactId>
+        <version>5.0.7-SNAPSHOT</version>
+    </parent>
+
+    <name>XChange Ftx Stream</name>
+    <artifactId>xchange-stream-ftx</artifactId>
+
+    <properties>
```

Comment:
I don't think we want to target java11 right now @timmolter ?

Reply:
i am sorry my bad, i use Java 11 and i didn't change it. I will change it and push again